.github/workflows/ci.yml

```
@@ -0,0 +1,206 @@
+        run: npm audit --audit-level=moderate
+
+      - name: Run tests
+        run: npm run test -- --run --coverage
+
+      - name: Build application
+        run: npm run build
+        env:
+          NODE_ENV: production
+
+      - name: Upload coverage reports
+        uses: codecov/codecov-action@v3
```

actionlint (1.7.7)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)